cmd/swapcli/main.go

```diff
@@ -422,7 +445,14 @@ func runMake(ctx *cli.Context) error {
 		}
 		defer wsc.Close()
 
-		id, statusCh, err := wsc.MakeOfferAndSubscribe(min, max, types.ExchangeRate(exchangeRate), ethAsset)
+		id, statusCh, err := wsc.MakeOfferAndSubscribe(
+			min,
+			max,
+			types.ExchangeRate(exchangeRate),
+			ethAsset,
+			relayerEndpoint,
+			relayerCommission,
```

Comment:
With 6 fields, we may want to create an input structure and make it the sole parameter to `MakeOfferAndSubscribe`.  Then you get labels on the values and, if valid, some values can be omitted. Just a thought, not required.